primo/data_parser/well_data_columns.py

```diff
@@ -75,6 +75,10 @@ class WellDataColumnNames:  # pylint: disable=too-many-instance-attributes
     # Columns for well_integrity metric
     well_integrity: Optional[str] = None
 
+    # Columns for efficiency metrics
+    elevation_delta: Optional[str] = None
+    dist_to_road: Optional[str] = None
```

Comment:
As mentioned above, please add `population_density` attribute.

Reply:
In a future PR.